.circleci/continue_config.yml

```diff
@@ -187,16 +222,19 @@ workflows:
   ci:
     jobs:
       - ci-base:
-          name: ci-base-<< matrix.python-version >>
+          name: ci-base-<< matrix.python-version >><< matrix.extras >>
```

Comment:
I would change this to `ci-base-<< matrix.python-version >>-<< matrix.extras >>` and remove the `-` from `extras` value. (or `ci-base-<< matrix.extras >>-<< matrix.python-version >>`, to me, display name is less of a concern compared with dealing with extra dash in the value)

note: references also need to be updated.

Reply:
Is this dangling hyphen a result of this change?

![Screen Shot 2023-06-07 at 9 47 45 AM](https://github.com/kolenaIO/kolena/assets/9648886/4200e3fd-974b-4c61-9492-c3cc67d5dae2)

Reply:
Apologies for the delay! Yes, because the extra-input is either `""` or `"metrics"` and the key is like `f"ci-base-{version}-{extra}"`

Reply:
I know that @ly-kolena has pointed in the other direction, but I'd like to avoid this dangling hyphen in these CI jobs. It's a low priority for sure